src/ray/rpc/grpc_client.h

```diff
@@ -49,8 +49,8 @@ class GrpcClient {
     // Disable http proxy since it disrupts local connections. TODO(ekl) we should make
     // this configurable, or selectively set it for known local connections only.
     argument.SetInt(GRPC_ARG_ENABLE_HTTP_PROXY, 0);
-    argument.SetMaxSendMessageSize(RayConfig::instance().max_grpc_message_size());
-    argument.SetMaxReceiveMessageSize(RayConfig::instance().max_grpc_message_size());
+    argument.SetMaxSendMessageSize(::RayConfig::instance().max_grpc_message_size());
```

Comment:
I noticed before that `RayConfig` is in the global namespace. Maybe we can move it to `ray::core` in this PR.